flow/project.py

```diff
@@ -2393,7 +2387,8 @@ def _verify_group_compatibility(self, groups):
 
     @contextlib.contextmanager
     def _potentially_buffered(self):
-        if self._use_buffered_mode:
+        """Enable the use of buffered mode for certain functions."""
+        if self.config['flow'].as_bool('use_buffered_mode'):
```

Comment:
The config is cached, so we don't need to have a separate private attribute `_use_buffered_mode` to store that. That attribute was only used in one place (this function).

Reply:
See previous comment.